### PR TITLE
Task 5

### PR DESCRIPTION
**Pull request by Oigen43**
All tasks completed 8/8

Link to application: https://d5szlxe6wew8g.cloudfront.net/
Link to FE PR: https://github.com/Oigen43/cloud-shop-web/pull/4/files

Screen how it work from CloudWatch
image

### CODE REVIEW

File: import-service/src/handlers/import-products-file/import-products-file.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    const { name } = event.queryStringParameters;
+    const path = `uploaded/${name}`;
+
+    const s3 = new AWS.S3({ region: 'us-east-1' });
+
+    const params = {
+      Bucket: 'import-products',
+      Key: path,
+      Expires: 60,
+      ContentType: 'text/csv',
```
Comment (helly-fox) [inline import-service/src/handlers/import-products-file/import-products-file.js:16]: Import is working incorrect (I can import whatever)

**Review by helly-fox — CHANGES_REQUESTED** [review]

File: import-service/src/handlers/import-products-file/import-products-file.js (hunk 1 above)
Comment (Oigen43) [reply import-service/src/handlers/import-products-file/import-products-file.js:16]: ok, i will fix it after crosscheck, but it is out of scope

Comment (helly-fox) [reply import-service/src/handlers/import-products-file/import-products-file.js:16]: Asking as seams like I do not think that your s3 settings are correct

Comment (Oigen43) [reply import-service/src/handlers/import-products-file/import-products-file.js:16]: what confuses you?

Comment (helly-fox) [reply import-service/src/handlers/import-products-file/import-products-file.js:16]: Sorry but smth is working definitly incorrect https://import-products.s3.amazonaws.com/uploaded/nike-just-do-it.jpg (uploaded folder at least should be empty)

Comment (helly-fox) [reply import-service/src/handlers/import-products-file/import-products-file.js:16]: Summarizing: for me seams like s3 bucket has public access as I can access files via link, and remove is not working as after uploading I can download uploaded file from uploaded folder

Comment (Oigen43) [reply import-service/src/handlers/import-products-file/import-products-file.js:16]: yes, you are right that I left open public access, now closed it

Comment (Oigen43) [reply import-service/src/handlers/import-products-file/import-products-file.js:16]: Screenshot from 2020-11-17 13-03-23
Screenshot from 2020-11-17 13-03-28
Screenshot from 2020-11-17 13-03-36

Comment (helly-fox) [reply import-service/src/handlers/import-products-file/import-products-file.js:16]: Thank you, now it is working

Comment (Oigen43) [reply import-service/src/handlers/import-products-file/import-products-file.js:16]: thank you for checking thoroughly

**Pull request merged**